templates/rpm/project.spec.erb

```diff
@@ -152,7 +175,18 @@ done
   <%- end -%>
 <%- end -%>
 
+
```

Comment:
@haus I added more whitespace to annoy you. Also, because it helps my brain to have a little more space between the script macros